README.md

```diff
@@ -216,14 +227,67 @@ contract LANDAuction is Ownable, LANDAuctionStorage {
     */
     function disableToken(address _address) public onlyOwner;
 
+    /**
+    * @dev Get exchange rate
+    * @param _srcToken - IERC20 token
+    * @param _destToken - IERC20 token
+    * @param _srcAmount - uint256 amount to be converted
+    * @return uint256 of the rate
+    */
+    function getRate(
+        IERC20 _srcToken,
+        IERC20 _destToken,
+        uint256 _srcAmount
+    ) public returns (uint256 rate)
+
     /**
     * @dev Convert allowed token to MANA and transfer the change in MANA to the sender
     * Note that we will use the slippageRate cause it has a 3% buffer
     * @param _fromToken - ERC20 token to be converted
     * @param _totalPrice - uint256 of the total amount in MANA
     * @return bool to confirm the convertion was successfully
     */
-    function convertSafe(ERC20 _fromToken, uint256 _totalPrice) internal returns (bool);
+    function _convertSafe(ERC20 _fromToken, uint256 _totalPrice) internal returns (bool);
+
+    /**
+    * @dev Validate bid function params
+    * @param _xs - uint256[] x values for the LANDs to bid
+    * @param _ys - uint256[] y values for the LANDs to bid
+    * @param _beneficiary - address beneficiary for the LANDs to bid
+    * @param _fromToken - token used to bid
+    */
+    function _validateBidParameters(
```

Comment:
What is the purpose of refactoring the requires into a function? I'm asking because I think it's a good practice to see the requires explicitly at the start of a function and I see this function is not reused elsewhere.

Reply:
IMO: It is hard to read large methods. Every `require` use > 1 line and we have 8 `require` which could be up to 10 regarding the pending features. Moving them to an internal function which takes care only of validating the params will free space and make semantic the steps of the method. 

If you read it now you will see:

1) validate params
2) transfer token or convert
3) transfer LAND

What do you think?

Reply:
My POV: In most other types of apps/code I would do the same thing. It just feels a bit odd when reading a contract, I typically don't expect a function that just does requires (except modifiers).

Reply:
Cool. So, are you ok to keep as it is?

Reply:
👍